src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts

```diff
@@ -11,6 +10,8 @@ export interface PaypalCommercePaymentInitializeOptions {
      * consumption.
      */
     form?: PaypalCommerceFormOptions;
+    style?: PaypalButtonStyleOptions;
+    submitForm?(): void;
```

Comment:
I'm not sure if it can be an optional parameter consider it is required in order to proceed (unless the shopper has started the flow elsewhere, i.e.: cart page).

Reply:
Hm, but we have credit cards and for them it's  optional parameter. And as you noticed, If the shopper approved account on the cart page.
And I added checks on this params [here](https://github.com/bigcommerce/checkout-sdk-js/pull/980/files#diff-3344e5b56907eaf188a8208fc12e5a51R48)